src/database/index.ts

```diff
@@ -0,0 +1,11 @@
+import createConnection from "./createConnection";
+
+const initializeDatabase = async (): Promise<void> => {
+  try {
+    await createConnection();
+  } catch (error) {
+    console.log(error);
+  }
+};
+
+export { initializeDatabase };
```

Comment:
You might not like this segregation... I needed to separate DB init and the server's one to avoid getting `initializeApp()` called in jest when it's run. So the solution was to make separate files for DB and server and orchestrate them in `index.ts` so that I could use them individually in a testing environment.
Another approach is to set DB and Server init in a single file so that init codes are less scattered around.

Reply:
it is alright!